README.md

```diff
@@ -77,9 +77,9 @@ as [Heroku](https://www.heroku.com/) to host the application.
 
 To try out the Broadcast Sample App, visit the following URLs:
```

Comment:
Let's reword this, you cant just navigate to the URL anymore, you need to launch the app someone first